ccpage114.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Start with the program you wrote for Exercise 6-1 (page 102).
+# Make two new dictionaries representing different people, and store all three dictionaries in a list called people.
+# Loop through your list of people . As you loop through the list, print everything you know about each person.
+friend_1 = {
+    'first_name': 'belina', 
+    'last_name': 'smith',
+    'city': 'boston',
+}
+
+
+friend_2 = {
+    'first_name': 'sammy',
+    'last_name': 'dallsby',
+    'city': 'san francisco',
+}
+
+friend_3 = {
+    'first_name': 'kelly',
+    'last_name': 'henderson',
+    'city': 'columbus',
+}
+
+people = [friend_1, friend_2, friend_3]
+
+for friend in people:
+    print(friend)
+
+
+# Make several dictionaries, where the name of each dictionary is the name of a pet.
+# In each dictionary, include the kind of animal and the owner’s name.
+# Store these dictionaries in a list called pets.
+# Next, loop through your list and as you do print everything you know about each pet.
+pets = {
+    'toby': {
+        'animal_type': 'dog',
+        'owner_name': 'posie',
+        },
+    
+    'elle': {
+        'animal_type': 'cat',
+        'owner_name': 'david',
+        },
+    
+    'kitty': {
+        'animal_type': 'hampster',
+        'owner_name': 'chris',
+        },
+}
+
+for pet_name, pet_info in pets.items():
+    print(f"\nPet name: {pet_name.capitalize()}")
```

Comment:
I'd recommend adding the line break at the _end_ (line 61), as opposed to the start. More conventional.